lib/migration.pm

```diff
@@ -93,7 +93,7 @@ sub register_system_in_textmode {
         register_addons_cmd();
     }
     else {
-        yast_scc_registration();
```

Comment:
I don't think this is needed, Yast detects the right interface to use automatically. Register_system is running in a text console. Do you have an example where we need it otherwise ?

Reply:
yes, I have two reasons to do this.

- please check the function name "register_system_in_textmode", obviously it required to run yast textmode(ncurses), I think it is necessary to do so in order to avoid the interference of other factors
- yes, I have examples on PowerVM like this ticket https://progress.opensuse.org/issues/119950, this is the openQA job https://openqa.suse.de/tests/9873489

Reply:
As my point of view, this looks like a workaround. In fact, I agree with Jonathon, 'Yast detects the right interface to use automatically.' So could we set the correct interface by default at the beginning of the test? So we don't need the workaround any more.

Reply:
Right, we should apply similar logic like in here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16035/files#diff-551e984de7a6559399feec270e9888674239837af9dbdf656091cd186418552dR218
probably makes sense to do inside the function and not at the level of test.